docs/modules/ROOT/pages/api/erc1155.adoc

```diff
@@ -1,10 +1,12 @@
 :github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
-:eip1155: https://eips.ethereum.org/EIPS/eip-1155[EIP1155]
+:eip1155: https://eips.ethereum.org/EIPS/eip-1155#metadata[EIP1155]
```

Comment:
i think we need to distinguish the base link and the metadata one, some places were linking to the base link like this one in the interface definition
<img width="774" alt="image" src="https://github.com/OpenZeppelin/cairo-contracts/assets/1703831/7b9c19a1-94eb-41e9-9f7f-45158762adb0">